### ISSUE TITLE
Port Baymed

### ISSUE THREAD

**Issue by Coureere** [discordia-space/CEV-Eris#3099]
What it says. We need to port a major feature to our build - brainmed/zurmed/gaymed and all other things that baymed might need like bay movement and else. Port is already started, and and other major features are dependent on that.

---
This is **anchor issue**. It mean it will be edited, new issues and PRs will be linked here, and additional information posted. This issue must be closed only after all of that will be done, and generally represent a rather bulky project.

**Comment by SpadesSlick** [comment]
Postponed for next time.

**Comment by ACCount12** [comment]
Baymed is dead, all hail Erismed!
